test/integration/client-side-encryption/driver.test.ts

```diff
@@ -526,11 +730,12 @@ describe('CSOT', function () {
     });
 
     const getKmsProviders = (): KMSProviders => {
-      const result = EJSON.parse(process.env.CSFLE_KMS_PROVIDERS || '{}') as unknown as {
-        local: unknown;
+      const { local } = EJSON.parse(process.env.CSFLE_KMS_PROVIDERS || '{}') as {
```

Comment:
Any reason not to use `getKmsProviders` in your new tests above? You currently have hard-coded a local key but we could get the provider from the environment.

Reply:
made this into a shared function